examples/tracing/otlp/README.md

```diff
@@ -65,3 +65,25 @@ service:
 
 1. Open the Datadog traces UI under the menu item on the left `APM` then `Traces`
 1. You should see a list of traces to explore
+
+### Clickhouse
```

Comment:
on second thought, im thinking this might not really belong here, as if I were looking for examples on how to do analytics with clickhouse, I probably wouldn't look in the `tracing/otlp` directory.

Maybe we can create a new `examples/analytics/clickhouse` directory and move all the clickhouse stuff and this part of this readme there. The fact that it uses OTLP is a bit of implementation detail IMO. WDYT?

Reply:
@markphelps Yeah good idea, I switched some things around.